Events/Ch26.event

```diff
@@ -0,0 +1,253 @@
+STAL 10
+MUSC 0x00
+STAL 40
+EARTHQUAKE_START 0x100
+STAL 40
+TILECHANGE 0x4
+STAL 10
+LOAD1 0x1 DZombie126
+ENUN
+SOUN 0x2f1
+STAL 30
+EARTHQUAKE_END
+MUSC 0x26 //Raid
+MOVE 0x5 0x91 [12,16]
+TILEREVERSE 0x4
+ENUN
+STAL 10
+//Text(ChTwentysixOpening3)
+STAL 30
+MUSC 0x00
+FADI 100
+//Text(ChTwentysixEnough)
+LOAD1 0x1 HellScene26
+ENUN
+FADU 100
+MUSC 0x2d //Lost Heart
+FlashCursor(Inferos,50)
+//Text(ChTwentysixHellDennis)
+//FIGH Inferos Dennis HellDennisFight26 0
+//KILL Dennis
+//find out how to kill units in FE8!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+DISA Dennis
+//Text(ChTwentysixHellVoid)
+MUSC 0x00
+FADI 100
+TILEREVERSE 0x2
+TILEREVERSE 0x3
+TILECHANGE 0x0
+//Text(ChTwentysixHellTeleport)
+//DISA Vedge
+
+DISA Inferos
+//LOAD2 0x1 VedgeReal26
+CHECK_HARD
+
+IFNOTEQUAL 0x8 0xC 0x0 //if the result of this check is not 0; i.e. true
+LOAD1 0x1 Bads26H
+ELSE 0x9
+LABEL 0x8
+LOAD1 0x1 Bads26
+LABEL 0x9
+CAM1 [12,7]
+MUSC 0x47 //wind ambience
+FADU 100
+//Text(ChTwentysixHellWelcome)
+FlashCursor(Vedge,50)
+
+MUSC 0x2f //Lights in the Dark; possibly change to His World if I'm not lazy
+STAL 20
+EARTHQUAKE_START 0x100
+STAL 50
+SOUN 0x11a //bolting strike
+TILEREVERSE 0x0
+STAL 20
+EARTHQUAKE_END
+MOVE 0x10 0x1 [12,9]
+FlashCursor(Inferos,50)
+//Text(ChTwentysixHellAngry)
+
+STAL 30
+GotoPrepScreen
+ENDA
+
+EndingScene26:
+MUSC 0x32 //victory
+
+FADI 5
+MNC2 0x1c //ch27? I have no idea what the ch number is at this point
+ENDA
+
+// Events
+
+PostPrep26:
+//Ersatz says something about the dzombie if he is deployed
+NoFade
+ENDA
+
+HellMoveback26:
+//if Inferos's battle quote was triggered, move him back
+//to his original place. Only Vedge can lure him out
+
+CHECK_EVENTID 0x1
+IFNOTEQUAL 0x2 0xC 0x0 //if the result of this check is not 0; i.e. true
+
+CHECK_ACTIVEID 0x1
+BNE 0x3 0x1 0xc //if active unit is not Vedge
+MOVE 0x10 Inferos [12,5]
+ENUF 0x1 //set 0x1 to false so this event can run again
+
+LABEL 0x3
+LABEL 0x2
+NoFade
+ENDA
+
+
+
+// Manual Movement
+
+// Scripted Fights
+
+//HellDennisFight26:
+//need to figure out how to do scripted fights in FE8!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+//DefenderMiss(0) 
+//AttackerCritical(60,1) 
+//BLDT
+
+// Units
+
+Bads26:
+UNIT Inferos EirikaMasterLord 0x00 Level(6, Enemy, 0) [12,5] 0x2 0x00 0x0 0x00 [FiendCleaver] AttackInRangeAI 
+UNIT 0x91 General_F Inferos Level(2, Enemy, 1) [8,17] 0x2 0x00 0x0 0x00 [Runesword] AttackInRangeAI 
+UNIT 0x91 General_F Inferos Level(2, Enemy, 1) [16,17] 0x2 0x00 0x0 0x00 [LightBrand] AttackInRangeAI 
+UNIT 0x91 Druid Inferos Level(2, Enemy, 1) [10,16] 0x00 0x00 0x0 0x00 [Luna, Physic] HealUnits 
+UNIT 0x91 Bishop Inferos Level(2, Enemy, 1) [14,16] 0x00 0x00 0x0 0x00 [Aura, Physic] HealUnits 
+//UNIT 0x91 Hero Inferos Level(2, Enemy, 1) [6,15] 0x00 0x00 0x0 0x00 [Halberd, FiendCleaver] AttackInRangeAI 
+//UNIT 0x91 Hero Inferos Level(2, Enemy, 1) [18,15] 0x00 0x00 0x0 0x00 [Armourslayer, ShadowKiller] AttackInRangeAI 

```

Comment:
Please do something with these commented units (either include them or remove them)

Reply:
Commented-out units were left in because I wasn't/am not sure how the unit count will stack up against someone who has been playing through 26 chapters of vidgame at this point.